lib/components/form/date-time-preview.js

```diff
@@ -83,12 +96,14 @@ class DateTimePreview extends Component {
             id: 'components.DateTimePreview.editDepartOrArrival'
           })}
           onClick={onClick}
+          style={{ padding: 0 }}
         >
           {editButtonText}
           {caret && (
             <span>
-              {' '}
-              <i className={`fa fa-caret-${caret}`} />
+              <StyledIconWrapper>
```

Comment:
With the removed `{' '}` does this still render correctly?

Reply:
It actually seems like this carat never gets used. I'm not really sure how I tested this before, I think I just put a "true" in the conditional renderer. If I put some text next to it it has decent spacing so I don't think it's necessary.

Reply:
It actually seems like this carat never gets used in the codebase. Maybe we just remove it? I'm not really sure how I tested this before, I think I just put a "true" in the conditional renderer. If I put some text next to it it has decent spacing so I don't think it's necessary.

Reply:
Will leave it up to the second reviewer to decide on this